app/default/props.conf

```diff
@@ -33,10 +33,26 @@ category = Structured
 description = Request logs produced by the Heroku HTTP routers.
 
 # Note: Only extract component as request logs contain a dyno field with a value that aligns with the other Heroku source types. Would always be "router" otherwise.
-EXTRACT-heroku_component = ^\S+\s(?<component>\S+)
+EXTRACT-component = ^\S+\s(?<component>\S+)
+
+# We convert `fwd` into a multivalue field using `fields.conf`.
+
+# Splunk CIM fields for the web model, see https://docs.splunk.com/Documentation/CIM/5.0.1/User/Web and https://devcenter.heroku.com/articles/http-routing#info-logs.
+# The `status` field already exists in the log message.
+EXTRACT-duration = service=(?<duration>\d+)ms
+EXTRACT-uri_path = path="(?<uri_path>[^#?"]+)\S*"
+EXTRACT-uri_query = path="\S+(?<uri_query>\?[^"]+)"
+EVAL-src = mvindex(fwd, 0)
+EVAL-url = protocol . "://" . host . path
+FIELDALIAS-http_method = method AS http_method
+FIELDALIAS-dest = host AS dest
+FIELDALIAS-bytes_out = bytes AS bytes_out
```

Comment:
Using `FIELDALIAS` to copy one existing field in to a new field with with a different name, the one the CIM is expecting!